src/source/Crypto/Dtls.h

```diff
@@ -29,15 +32,16 @@ extern "C" {
 
 #define SECONDS_IN_A_DAY                    (24 * 60 * 60LL)
 
-#define LOG_OPENSSL_ERROR(s)                    while ((sslErr = ERR_get_error()) != 0) { \
-                                                    if (sslErr != SSL_ERROR_WANT_WRITE && sslErr != SSL_ERROR_WANT_READ) { \
-                                                        DLOGW("%s failed with %s", (s), ERR_error_string(sslErr, NULL)); \
-                                                    } \
-                                                }
+#define HUNDREDS_OF_NANOS_IN_A_DAY          (HUNDREDS_OF_NANOS_IN_AN_HOUR * 24LL)
 
 typedef enum {
+#ifdef KVS_USE_OPENSSL
```

Comment:
Minor. These could be outsourced to Crypto.h that you've created. Alias them like KVS_SRTP_AES128_CM_SHA1_80 or something. Not a biggie

Reply:
Yep, makes sense!